supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -57,6 +57,17 @@
 
 *See also*:
 
+[[peer-pod]]
+==== image:images/yes.png[yes] peer pod (noun)
+*Description*: In Red{nbsp}Hat OpenShift sandboxed containers, a _peer pod_ extends the concept of a standard pod. Unlike a standard sandboxed container, where the virtual machine is created on the worker node itself, in a peer pod, the virtual machine is created through a remote hypervisor using any supported hypervisor or cloud provider API.
+
+*Use it*: yes
+
+[.vale-ignore]
```

Comment:
I don't understand why we want Vale to ignore the incorrect form? Is the incorrect form allowed in some circumstances?

Reply:
We do this so that Vale doesn't flag all of these incorrect forms as incorrect when it runs on the SSG repo. They are incorrect and we know them and want them. We want Vale to flag only items that are used incorrectly elsewhere in the SSG.